src/snapshot/index.js

```diff
@@ -34,7 +34,8 @@ var Snapshot = {
     clone: require('./cloneplot'),
     toSVG: require('./tosvg'),
     svgToImg: require('./svgtoimg'),
-    toImage: require('./toimage')
+    toImage: require('./toimage'),
+    toSnapshot: require('./tosnapshot')
```

Comment:
actually, `tosnapshot` remains from my first misguided attempt.  I can eliminate, since I'm not sure it has much value any more with our improved promise-based `toimage`.